JSDemos/Demos/Toast/Overview/description.md

```diff
@@ -1 +1,25 @@
-The Toast is&nbsp;a&nbsp;UI component that provides pop-up notifications. It&nbsp;can be&nbsp;displayed in&nbsp;one of&nbsp;the following modes: _&laquo;info&raquo;_, _&laquo;warning&raquo;_, _&laquo;error&raquo;_, _&laquo;success&raquo;_ and _&laquo;custom&raquo;_. In&nbsp;this demo, the _&laquo;success&raquo;_ and _&laquo;error&raquo;_ modes are used depending on&nbsp;the context.
\ No newline at end of file
+The Toast is a UI component that provides pop-up notifications. 
```

Comment:
Just be careful with the word "provides": https://tw.devexpress.devx/language-guidelines/overused-words/provide/

```suggestion
The Toast is a UI component that displays pop-up notifications. 
```